core/aws_ddk_core/stages/appflow_lambda/index.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Is `index.py` a convention for lambda handlers in python? Also, could we have something along the lines of `lambda_handlers/appflow_check_flow_status/` with `lambda_handlers` folder for other handlers in the future

Reply:
Moved to `lambda_handlers/appflow_check_flow_status/` as suggested. As for the name of the file, [by default](https://docs.aws.amazon.com/lambda/latest/dg/python-handler.html#naming) it's using `lambda_function.py`. Perhaps I should use a more explicit name, like `appflow_flow_status.py`?

Reply:
Convention is `lambda_function.lambda_handler` so I think we can stick with that

Reply:
renamed!